lib/ui/app/templates/components/ui/empty-message.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ui/templates/components/empty-message';
```

Comment:
Why are we moving all these files?

Reply:
This is related to #1101, but now superseded by #1102. Once that is merged I'll rebase this to drop that commit.